torch/csrc/jit/codegen/cuda/executor_launch_params.h

```diff
@@ -92,7 +92,6 @@ class TORCH_CUDA_CU_API LaunchParams {
     if (class_val == UNINITIALIZED_VAL) {
       class_val = incoming_val;
     }
-    assertValid();
```

Comment:
Is this removed because the shapes are validated somewhere else?

Reply:
Yes, `LaunchParams::bind` calls `assertValid()` twice, which shouldn't be necessary and also `assertValid` does take some time.